### PR TITLE
AF-1229: Cover NumericIntegerTextBox

### PR DESCRIPTION
**Pull request by jomarko**
This PR is a corollary of the selenium test migration into community, for more details see the:
https://issues.jboss.org/browse/AF-1229

### CODE REVIEW

**Comment by jomarko** [comment]
@ederign please re-review.

File: uberfire-extensions/uberfire-widgets/uberfire-widgets-commons/src/test/java/org/uberfire/ext/widgets/common/client/common/NumericIntegerTextBoxTest.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
+package org.uberfire.ext.widgets.common.client.common;
```
Comment (manstis) [inline uberfire-extensions/uberfire-widgets/uberfire-widgets-commons/src/test/java/org/uberfire/ext/widgets/common/client/common/NumericIntegerTextBoxTest.java]: License please.

Comment (jomarko) [reply uberfire-extensions/uberfire-widgets/uberfire-widgets-commons/src/test/java/org/uberfire/ext/widgets/common/client/common/NumericIntegerTextBoxTest.java]: Done, thanks.

**Comment by ederign** [comment]
Waiting for the green...

**Comment by jomarko** [comment]
jenkins please retest this

**Comment by jomarko** [comment]
@manstis @ederign it is green now.

**Comment by ederign** [comment]
@jomarko thanks!

**Pull request merged**